vhost-device-i2c/src/i2c.rs

```diff
@@ -369,8 +370,11 @@ impl PhysDevice {
 }
 
 impl I2cDevice for PhysDevice {
-    fn open(adapter_name: &str) -> Result<Self> {
-        let adapter_no = PhysDevice::find_adapter(adapter_name)?;
+    fn open(adapter_identifier: &AdapterIdentifier) -> Result<Self> {
+        let adapter_no = match adapter_identifier {
+            AdapterIdentifier::Name(adapter_name) => PhysDevice::find_adapter(adapter_name)?,
+            AdapterIdentifier::Number(no) => *no as u32,
+        };
         let device_path = format!("/dev/i2c-{}", adapter_no);
```

Comment:
If a new method like `get_no()` were added to the `AdapterIdentifier` , under the adapter_name case, it might be necessary to move `find_adapter()` into `AdapterIdentifier` in order to determine the actual I2C number. This way, the `PhysDevice` would not need to worry about how to find the adapter by name. That makes sense.

Reply:
It seems that `find_adapter()` is an exclusive function of `PhysDevice`. 
Whether is it necessary to move it to `AdapterIdentifier` ?

Reply:
I don't see any issue in marking that function `pub`.

BTW I don't have a strong opinion, reviewing the code seemed like the reasonable thing to do, but it's a small thing, you can leave it that way if you prefer.

Reply:
Thanks @stefano-garzarella 
I prefer to keep it as it is.